keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/AdminController.kt

```diff
@@ -0,0 +1,29 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.persistence.DeliveryConfigRepository
+import com.netflix.spinnaker.keel.persistence.ResourceRepository
+import org.slf4j.LoggerFactory.getLogger
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/admin"])
+class AdminController(
+  private val deliveryConfigRepository: DeliveryConfigRepository,
+  private val resourceRepository: ResourceRepository
+) {
+  private val log by lazy { getLogger(javaClass) }
+
+  @DeleteMapping(
+    path = ["/{application}"]
+  )
```

Comment:
This should probably have `@ResponseStatus(ACCEPTED)` here, and not return anything.